SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/SpatialRPCService.cpp

```diff
@@ -530,6 +530,31 @@ FRPCErrorInfo SpatialRPCService::ApplyRPC(const FPendingRPCParams& Params)
 	return ApplyRPCInternal(TargetObject, Function, Params);
 }
 
+struct NetWriteFenceResolutionHandler
+{
+	NetWriteFenceResolutionHandler(USpatialNetDriver& InNetDriver, UFunction& Function)
+		: NetDriver(InNetDriver)
+		, bIsNetWriteFence(Function.HasAnyFunctionFlags(FUNC_NetWriteFence))
+	{
+		if (bIsNetWriteFence)
+		{
+			NetDriver.PushNetWriteFenceResolution();
+		}
+	}
+
+	~NetWriteFenceResolutionHandler()
```

Comment:
non-virtual, I would make this struct 'final'.

Reply:
-> Fixed with FStackOnly